_gcode/M043.md

```diff
@@ -12,12 +12,11 @@ codes:
   - M43
 
 long:
-  - When setting up or debugging a machine it's useful to know how pins are assigned to functions by the firmware, and to be able to find pins for use with new functions. `M43` provides these tools. `M43` by itself reports all pin assignments. Use `P` to specify a single pin. Use `W` to watch the specified pin, or all pins. Use the `L` option to monitor endstops (Note: use `E` in firmware prior to 1.1.7).
+  - When setting up or debugging a machine it's useful to know how pins are assigned to functions by the firmware, and to be able to find pins for use with new functions. `M43` provides these tools. `M43` by itself reports all pin assignments. Use `P` to specify a single pin. Use `I` to report the values on pins that are protected. Use `W` to watch the specified pin, or all pins. Use the `E` option to monitor endstops. Use `S` option to test a BLTouch type servo probe.  Use `T` option to toggle pins.
   - The `W` watch mode option continues looping, blocking all further commands, until the board is reset. If `EMERGENCY_PARSER` is enabled, `M108` may also be used to exit the watch loop without needing to reset the board.
 
 notes:
   - Requires `PINS_DEBUGGING`. This feature should be disabled for production use.
-  - As of 1.1.7, the `E` parameter has been changed to `L` to match the behavior of Repetier Host.
```

Comment:
Are we no longer matching behavior with Repetier, or…?

Reply:
I don't know why the current code has "E".  If you want i can change the code and the M43 docs to "L".

There isn't a link to the M43 T index in the main M43.  I'll add that.